components/AppHeader2/wrapper.tsx

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useRef, useState } from 'react';
+
+import clsx from 'clsx';
+import AppHeader2_Core from './core';
+
+export const APP_HEADER_HEIGHT = 86;
+const INITIAL_HEADER_HEIGHT = APP_HEADER_HEIGHT;
+const TOP_OFFSET = 0; // Can be set to APP_HEADER_HEIGHT if want to app bar to be separated from the Hero part
+
+export default function AppHeader2_Wrapper() {
```

Comment:
Thoughts on making the navbar sticky on top?

Reply:
Yeah I can do that. So no more hiding?